### ISSUE TITLE
Can you use WebRTC on naitive platforms?

### ISSUE THREAD

**Issue by shinichihezemans** [webrtc-rs/webrtc#439]
I am relatively new to WebRTC and im wondering if i could use this project in a naitive application without using a browsers or a headless browser. Like if i want to set up a peer data connection between a linux application and a windows application. I have been trying to use the examples but a haven't been able to run them correctly. Are there any resources i could look at, i haven't been able to find a lot online.

**Comment by DougAnderson444** [comment]
Yes you can. In fact, this library is meant for native applications (such as a WebRTC Server) and cannot be used in the browser.
